src/kubernetes.js

```diff
@@ -35,4 +35,97 @@ export class KubernetesChaos {
     killNamespace(namespaceName) {
         return this.client.namespaces.delete(namespaceName)
     }
-}
\ No newline at end of file
+
+class JobRunner {
+
+    constructor(client) {
+        this.client = client
+    }
+
+   runJobOnNodes(namespace, jobName, nodes, image, command){
+        let jobs = []
+	for (let i = 0; i < nodes.length; i++) {
+          let name = `${jobName}-${(Math.random() + 1).toString(36).substring(7)}`;
+          console.log(`Creating job ${name} on node ${nodes[i]}`)
+          const job = this.client.jobs.create({
+            namespace: namespace,
+            name: name,
+            image: image,
+            command: command,
+            node_name: nodes[i]
+          })
+	  jobs.push(name)
+        }
+        return jobs
+    }
+
+    cleanJobs(namespace, jobs){
+        for (let i = 0; i < jobs.length; i++) {
+           console.log(`Deleting job ${jobs[i]}`)
+           this.client.jobs.delete(jobs[i],namespace)
+        }
+        return jobName
+    }
+
+}
+
+export class StressNodesAttack {
+    constructor(client) {
+        this.client = client
+	this.jobRunner = new JobRunner(client)
+        this.name = 'stress-nodes'
+        this.namespace = 'default'
+        this.cpu = '0'
+        this.io = '0'
+        this.vm = '0'
+        this.vmBytes = '256MB'
+        this.timeout = '10'
+    }
+
+    inNamespace(namespace) {
+        this.namespace = namespace
+        return this
+    }
+    inNodes(nodes) {
+        this.nodes = nodes
+        return this
+    }
+    withName(name) {
+        this.name = name
+        return this
+    }
+    withCPU(cpu) {
+        this.cpu = cpu
+        return this
+    }
+    withIO(io) {
+        this.io = io
+        return this
+    }
+    withVM(vm) {
+        this.vm = vm
+        return this
+    }
+    withVMBytes(vmBytes) {
+        this.vmBytes = vmBytes
+        return this
+    }
+    withTimeout(timeout) {
+        this.timeout = timeout
+        return this
+    }
+    start() {
+        this.jobs = this.jobRunner.runJobOnNodes(
+            this.namespace,
+            this.name,
+            this.nodes,
+            'progrium/stress',
```

Comment:
From what I've seen, this tool is no longer maintained. We maybe should migrate to use [stress-ng](https://wiki.ubuntu.com/Kernel/Reference/stress-ng). We can do this in a follow up MR.

Reply:
Yeah! Also, I think we could maintain the Docker image ourselves. Probably we can pack other tools in there.